### ISSUE TITLE
Lost Notes

### ISSUE THREAD

**Issue by your-acc** [cabalex/fnf-song-converter#15]
When I used the chart converter, the beginning part where the player doesn't repeat the notes gets lost and is (almost) identical to the original, even in the pixel part where it's all default!
Don't believe me? type "73" in song selector here:
https://scratch.mit.edu/projects/631532535/

**Comment by your-acc** [comment]
Video form if you don't want to/can't view it;
https://user-images.githubusercontent.com/82058673/157147028-f14efc7c-c64e-45cf-a910-b56bf65ea43e.mp4

**Comment by cabalex** [comment]
Do you have the original chart JSON file you used to convert? I'd love to take a look and see what's going on.

**Comment by your-acc** [comment]
Right here!
[you-cant-run-hard.txt](https://github.com/cabalex/fnf-song-converter/files/8202637/you-cant-run-hard.txt)

**Comment by your-acc** [comment]
Wrong one.
[scratch friendly!.txt](https://github.com/cabalex/fnf-song-converter/files/8202649/scratch.friendly.txt)

**Comment by your-acc** [comment]
You can tell I get everything wrong by two deleted messages.
[New Text Document.txt](https://github.com/cabalex/fnf-song-converter/files/8202654/New.Text.Document.txt)

**Comment by cabalex** [comment]
Found the issue! This song uses an abnormal amount of data for the notes (5 total data fields per note compared to the 3 in vanilla). Changing extended note handling to "allow all notes (experimental)" would have fixed the issue.

However, this adds some extra data that the Scratch base engine doesn't know how to handle [note length breaks], so I've added some extra truncating options to the converter. See [the wiki](https://github.com/cabalex/fnf-song-converter/wiki/Home#what-do-the-extended-note-handling-options-mean) for more information, but for your use case you would set it to **Allow all notes, truncate extra data (EXPERIMENTAL)**.
